sdk/scheduler/src/test/java/com/mesosphere/sdk/state/PersistentLaunchRecorderTest.java

```diff
@@ -29,7 +29,8 @@
             .build();
 
     @BeforeClass
-    public static void beforeAll() throws Exception {
+    public static void beforeAll2() throws Exception {
+        // Use a new beforeAll name to avoid shadowing parent final void
```

Comment:
- May make sense to just rename the one in OfferEvaluatorTestBase instead. Could see this coming up with other extending classes
- Comment should probably be in javadoc form (`/** foo */`)